### DISCUSSION TITLE
Assigning material

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#255] [General]
In example/tensile_test, how can I assign material in different layer of tensile_test sample of same block id with the help of coreform cubit.

**Comment by oldninja** [comment]
I don't think you can do that. You will need to create different blocks for the different layers, if you want to do this. I would try to slice the tensile specimen in a way that mimics what you want to model for any layers.

**Reply by Nikhil23121998** [reply]
can I slice the sample with the help of toggle clipping plane tool and assign different material to the different section of sample.

**Reply by oldninja** [reply]
Yes. Just make sure they are different "blocks" in exodus format. I've done this, but using the text-based input format for Peridigm.

**Reply by Nikhil23121998** [reply]
I am doing it in genesis format, and after running it with the help of YAML file to generate exodus file, but when i am increasing number of block it is showing error.
